mods/cnc/maps/nod02b/nod02b.lua

```diff
@@ -139,22 +139,22 @@ WorldLoaded = function()
 end
 
 Tick = function()
-	if Nod.HasNoRequiredUnits() then
+	if player.HasNoRequiredUnits() then
 		if DateTime.GameTime > 2 then
-			GDI.MarkCompletedObjective(GDIObjective)
+			enemy.MarkCompletedObjective(GDIObjective)
 		end
 	end
 
-	if GDI.HasNoRequiredUnits() then
-		Nod.MarkCompletedObjective(NodObjective2)
+	if enemy.HasNoRequiredUnits() then
+		player.MarkCompletedObjective(NodObjective2)
 	end
 
-	if DateTime.GameTime % DateTime.Seconds(1) == 0 and not Nod.IsObjectiveCompleted(NodObjective1) and CheckForBase(Nod) then
-		Nod.MarkCompletedObjective(NodObjective1)
+	if DateTime.GameTime % DateTime.Seconds(1) == 0 and not player.IsObjectiveCompleted(NodObjective1) and CheckForBase(player) then
+		player.MarkCompletedObjective(NodObjective1)
 	end
 
-	if DateTime.GameTime % DateTime.Seconds(3) == 0 and Barracks.IsInWorld and Barracks.Owner == gdi then
-		checkProduction(GDI)
```

Comment:
units wouldn't get produced since the lowercase "gdi" would always fail the comparison (playername was "GDI")